src/app/games/siam/tests/siam.component.spec.ts

```diff
@@ -272,4 +271,55 @@ describe('SiamComponent', () => {
         const move: SiamMove = SiamMove.from(5, 4, MGPOptional.of(Orthogonal.LEFT), Orthogonal.LEFT).get();
         await testUtils.expectMoveSuccess('#indicator_4_4_LEFT', move);
     }));
+
+    function expectTranslationYToBe(transform: SVGTransform, y: number): void {
+        expect(transform.type).toBe(SVGTransform.SVG_TRANSFORM_TRANSLATE);
+        // In a SVG transform, f is the y coordinate
+        expect(transform.matrix.f).toBe(y);
+    }
+    it('should display current player pieces on the bottom (Player.ONE)', () => {
+        // Given a state from the point of view of player 1
+        const board: Table<SiamPiece> = [
+            [_, _, _, _, _],
+            [_, _, _, _, _],
+            [_, M, M, M, _],
+            [_, _, _, _, _],
+            [_, _, _, _, U],
+        ];
+        const state: SiamState = new SiamState(board, 1);
+
+        // When the game is displayed
+        testUtils.setupState(state);
+
+        // Then player 1's pieces should be on the bottom
+        const playerOneRemainingPiecesTranslation: SVGTransform =
+            testUtils.findElement('#remainingPieces_1_0').nativeElement.transform.baseVal.getItem(0);
+        const playerZeroRemainingPiecesTranslation: SVGTransform =
+            testUtils.findElement('#remainingPieces_0_0').nativeElement.transform.baseVal.getItem(0);
+        expectTranslationYToBe(playerZeroRemainingPiecesTranslation, -100)
+        expectTranslationYToBe(playerOneRemainingPiecesTranslation, 700)
+    });
+    it('should display current player pieces on the bottom (observer)', () => {
```

Comment:
bah y a pas de currentPlayer là du coup, c'est zero qui doit être en dessous

Reply:
exact, c'est le commentaire + it qu'était pas bon, fixé

Reply:
Mais la phrase "should display current player pieces on the botton (observer)" représente pas la réalité souhaitée non plus !

Reply:
changé en "should display player zero pieces [...]"